packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -41,10 +41,17 @@ const isObjectEmpty = (obj: unknown) => {
   return Object.keys(obj).length === 0;
 };
 
-export default function useFeedSettings(): FeedSettingsReturnType {
+interface UseFeedSettingsProps {
+  enabled?: boolean;
+}
+
+const FEED_SETTINGS_STALE_TIME = 1000 * 60;
```

Comment:
Maybe put it where we have the react query key generator fn?
And do something like:
```
const ONE_MINUTE = 1000 * 60
export const FEED_SETTINGS_STALE_TIME = ONE_MINUTE
```

(will allow us to be more flexible in the future. (and maybe define: DEFAULT_STALE_TIME = X_MINUTE

Reply:
Yeah, that's better.